database/migrations/2022_03_25_133443_create_intializer_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateIntializerTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
Create your table in such a way that it incorporates all the PSPs and when need is, you select for the specific PSP the fields that correspond to it and use.